worked on creating paragraph structure and fixing ocr errors

cgguerra opened the pull request.

cgguerra committed:
worked on creating paragraph structure and fixing ocr errors

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/scta-texts/lombardsententia?pullRequest=5) <br/>All committers have signed the CLA.

cgguerra committed:
practice changes at our meeting

cgguerra committed:
First changes! I had a few errors

cgguerra committed:
finished distribution 42, but some errors and questions

cgguerra committed:
Update pl-l2d43c1.xml

cgguerra committed:
new section that needs review

I have some questions about some errors that keep coming up today.

jeffreycwitt commented on pl-l2d42c8/pl-l2d42c8.xml:
```diff
@@ -59,15 +59,15 @@
-
+        <ref>
           <name>Augustinus<name>
+        <ref>
```
@cgguerra notice here that your second `<ref>` on code line 64 should be a closing `</ref>`. So it should have a slash in it, like so : `</ref>`

jeffreycwitt commented on pl-l2d42c8/pl-l2d42c8.xml:
```diff
@@ -59,15 +59,15 @@
-          excellere nisi putaret, per hoc maiores divitias habere.
-            <quote>
+        <quote>
+          , homo, qui non esset amator pecuniae, nisi per hoc putaret, se excellentiorem esse, 
```
`<quote></quote>` should be wrapped in a `<cit></cit>` element. 

So this should be `<cit><quote>text....</quote></cit>`

jeffreycwitt replied:
the comma before "homo" is not really part of the quote. So this should really look like the following: 

```
<ref>Augustinus</ref>,
<cit>
   <quote>
     text text
   </quote>
</cit>
```

jeffreycwitt commented on pl-l2d43c1/pl-l2d43c1.xml:
```diff
@@ -276,47 +276,86 @@
           carne vivit, non potest iudicari. De nullo enim desperandum est, quamdiu patientia Dei ad
-          poenitentiam adducit. Paganus est hodie, 
+          poenitentiam adducit. Paganus est hodie,
+          <quote>
```
closing slash in close `</quote>` is missing. (You're editor should be giving you red errors when you do this; we can check on Thursday to make sure that is working)

jeffreycwitt commented on pl-l2d43c1/pl-l2d43c1.xml:
```diff
@@ -276,47 +276,86 @@
+
+        <p xml:id="pl-l2d43c1-d1e153"> De hoc quoque peccato in Spiritum sanctum 
+        <ref>
+          <name>Ambrosius 
```
`<name>` here is missing a closing tag
(and generally all of your closing tags are missing the slash)

jeffreycwitt commented on pl-l2d43c1/pl-l2d43c1.xml:
```diff
@@ -276,47 +276,86 @@
+          <ref>
+          <cit>
+            <quote>
+              , non potest ibi exhortatio
```
check comma placement. See my comment above.

jeffreycwitt commented on pl-l2d44c1/pl-l2d44c1.xml:
```diff
@@ -54,41 +54,93 @@
+        <p xml:id="pl-l2d44c1-d1e120"> Ait enim 
+        <ref>
+          <name>Apostolus<name>
+          <!--this is a reference to Paul since it's quoting Romans, but not sure if it deserves a name tag-->
```
yes `<name>` tag inside `<ref>` is correct here, but `Apostolus` is a special way they refer to the apostle paul. When the say "the apostle", they always mean Paul

jeffreycwitt commented on pl-l2d44c1/pl-l2d44c1.xml:
```diff
@@ -54,41 +54,93 @@
+         ; quod non de: potestate
+          boni tantum, sed et mali intelligi oportet, cum 
+          <name>Pilato<name> 
+          <!--same question as before, do Biblical figures get a name tag? -->
```
yes, this is perfect.

jeffreycwitt commented on pl-l2d44c2/pl-l2d44c2.xml:
```diff
@@ -54,21 +54,44 @@
-          in tributis et huiusmodi. Si vero princeps aliquis vel diabolus aliquid e iusse- rit vel
+        
+        <p xml:id="pl-l2d44c2-d1e108"> Hic oritur quaestio non transilienda silentio. Dictum est enim supra, quod potestas
+          peccandi et nocendi non est homini vel dia-bolo nisi a Deo. 
```
dash inside dia-bolo should be removed.

jeffreycwitt commented on pl-l2d44c2/pl-l2d44c2.xml:
```diff
@@ -54,21 +54,44 @@
+        <cit>
+           Cum ergo diabolo sit potestas mali,
+          Dei ordinatione, potestati eius non esse resistendum videtur. </p>
+          <!--I've gotten this error several times in other files-->
```
you're getting errors everywhere because you're not "closing" elements correctly. Once you fix this, most of your errors will resolve.

cgguerra committed:
what we worked on today in our hour

cgguerra committed:
Merge branch 'scta-texts:master' into master

cgguerra committed:
Merge branch 'master' of https://github.com/cgguerra/lombardsententia

cgguerra committed:
I haven't looked at the Lombard for these. I just cleaned everything up so I can read and make corrections to the Latin next week.

cgguerra committed:
I fixed the spacing and checked the Latin again.

There are a few things I have questions about, which I have marked with comments.

cgguerra committed:
I had lots of names inside quotes, so I made comments about problem areas

cgguerra committed:
Can we look at names inside quotes on Thursday?

I keep having names appear inside quotes. Could you please show me how to properly include names inside a quote on Thursday? I have made comments every time I have come across this problem.

cgguerra committed:
new things

cgguerra committed:
SCTA work today

cgguerra committed:
Fixed references

I need to check some of the Lombard book 2 quotes again.
